app/connector/support/maven-plugin/pom.xml

```
@@ -75,6 +90,16 @@
       <artifactId>jackson-databind</artifactId>
     </dependency>
 
+    <dependency>
+        <groupId>com.github.fge</groupId>
+        <artifactId>json-schema-core</artifactId>
+    </dependency>
+
+    <dependency>
+        <groupId>com.github.fge</groupId>
+        <artifactId>json-schema-validator</artifactId>
+    </dependency>
```

See above, just add version here. Might want to format on two spaces.
```suggestion
      <version>2.2.6</version>
    </dependency>
```